### PR TITLE
[ISSUE #7199] grpcClientChannel header add null judgement

### PR DESCRIPTION
**Pull request by weihubeats**
https://github.com/apache/rocketmq/issues/7199

### CODE REVIEW

File: proxy/src/main/java/org/apache/rocketmq/proxy/grpc/v2/channel/GrpcClientChannel.java [hunk 1]
```diff
@@ -210,7 +211,7 @@ protected CompletableFuture<Void> processCheckTransaction(CheckTransactionStateR
     protected CompletableFuture<Void> processGetConsumerRunningInfo(RemotingCommand command,
         GetConsumerRunningInfoRequestHeader header,
         CompletableFuture<ProxyRelayResult<ConsumerRunningInfo>> responseFuture) {
-        if (!header.isJstackEnable()) {
+        if (Objects.isNull(header) || !header.isJstackEnable()) {
```
Comment (joeCarf) [inline proxy/src/main/java/org/apache/rocketmq/proxy/grpc/v2/channel/GrpcClientChannel.java:214]: can you explain why should this func just return when `header` is null?

Comment (weihubeats) [reply proxy/src/main/java/org/apache/rocketmq/proxy/grpc/v2/channel/GrpcClientChannel.java:214]: Failure to return will result in a null pointer error causing a large error log to be printed.

**Comment by weihubeats** [comment]
@Oliverwqcwrw Can you look at this pr?

**Review by Oliverwqcwrw — APPROVED** [review]
Although I don't think the header should be null in theory, it is necessary for the robustness of the code to add this null judgment.

**Comment by weihubeats** [comment]
@RongtongJin Can you help merge this pr

**Comment by weihubeats** [comment]
@Oliverwqcwrw Can you help merge this pr

**Comment by weihubeats** [comment]
@RongtongJin  Can you help merge this pr

**Review by joeCarf — APPROVED** [review]

**Comment by weihubeats** [comment]
@Oliverwqcwrw Can you look at this pr?

**Comment by Oliverwqcwrw** [comment]
> @Oliverwqcwrw Can you look at this pr?

Thanks for your contribution

**Pull request merged**